### PR TITLE
46586-4_22-reject-evidence-note-transfer-hint-wordings

### PR DESCRIPTION
**Pull request by GuidoWorkDev**
amended reject/return hint text

### CODE REVIEW

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewEvidenceNotesViewModelMapTests.cs [hunk 1]
```diff
@@ -134,9 +134,11 @@ public void Map_GivenValidEvidenceNoteData_ShouldContainHintItems()
             result.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note");
             result.HintItems.ElementAt(0).Value.Should().BeNull();
             result.HintItems.ElementAt(1).Key.Should().Be("Reject evidence note");
-            result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
+            //result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewEvidenceNotesViewModelMapTests.cs:137]: @GuidoWorkDev  Remove all of the commented out code

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewTransferNoteViewModelMapTests.cs [hunk 2]
```diff
@@ -111,9 +111,11 @@ public void Map_GivenValidEvidenceNoteData_ShouldContainHintItems()
             result.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note transfer");
             result.HintItems.ElementAt(0).Value.Should().BeNull();
             result.HintItems.ElementAt(1).Key.Should().Be("Reject evidence note transfer");
-            result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
+            //result.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewTransferNoteViewModelMapTests.cs:114]: @GuidoWorkDev  Remove all of the commented out code

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewEvidenceNoteViewModelTests.cs [hunk 3]
```diff
@@ -83,9 +83,11 @@ public void ReviewEvidenceNoteViewModel_ShouldBeInitialisedWithHintItems()
             //assert
             model.HintItems.ElementAt(0).Value.Should().Be(null);
             model.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note");
-            model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
+            //model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note");
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewEvidenceNoteViewModelTests.cs:86]: @GuidoWorkDev  Remove all of the commented out code

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewTransferNoteViewModelTests.cs [hunk 4]
```diff
@@ -81,9 +81,11 @@ public void ReviewEvidenceNoteViewModel_ShouldBeInitialisedWithHintItems()
             //assert
             model.HintItems.ElementAt(0).Value.Should().Be(null);
             model.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note transfer");
-            model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
+            //model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewTransferNoteViewModelTests.cs:84]: @GuidoWorkDev  Remove all of the commented out code

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewEvidenceNotesViewModelMapTests.cs (hunk 1 above)
Comment (GuidoWorkDev) [reply src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewEvidenceNotesViewModelMapTests.cs:137]: whoopsey

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewEvidenceNoteViewModelTests.cs (hunk 3 above)
Comment (GuidoWorkDev) [reply src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewEvidenceNoteViewModelTests.cs:86]: idem

File: src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewTransferNoteViewModelTests.cs (hunk 4 above)
Comment (GuidoWorkDev) [reply src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewTransferNoteViewModelTests.cs:84]: run-out of adjectives

**Pull request merged**